contributing/code/conventions.rst

```diff
@@ -103,4 +103,9 @@ A PHP ``E_USER_DEPRECATED`` error must also be triggered to help people with
 the migration starting one or two minor versions before the version where the
 feature will be removed (depending on the criticality of the removal)::
 
-    trigger_error('XXX() is deprecated since version 2.X and will be removed in 2.Y. Use XXX instead.', E_USER_DEPRECATED);
+    @trigger_error('XXX() is deprecated since version 2.8 and will be removed in 3.0. Use XXX instead.', E_USER_DEPRECATED);
+
+Without the @-silencing operator, users would need to opt-out from deprecation
```

Comment:
Apparently, the `@-silencing operator` is officially called by PHP the `error-control operator` or the `@-operator`. See http://php.net/manual/en/language.operators.errorcontrol.php  Maybe we can keep the `@-silencing operator` but adding a link to the official documentation?